addons/survey/views/survey_templates_statistics.xml

```diff
@@ -67,29 +66,31 @@
         <div class="o_survey_results_question pb-5 border-bottom">
         <div class="d-flex align-items-start mb-2">
             <div class="mr-auto">
-                <h5 t-field="question.title" class="mb-3"/>
+                <h5 t-field="question.title" class="mb-3 mr-3"/>
             </div>
             <!-- Question info -->
-            <span class="badge badge-info" t-field='question.question_type'/>
-            <t t-if="question.question_type == 'matrix'">
-                <span class="badge badge-info ml-2" t-field='question.matrix_subtype'/>
-            </t>
-            <!-- Scoring info -->
-            <t t-if="question.is_scored_question">
-                <span class="badge badge-success ml-3"><span t-esc="question_data['right_inputs_count']" class="mr-1"/>Correct</span>
-                <t t-if="question.question_type in ['simple_choice', 'multiple_choice']">
-                    <span class="badge badge-warning ml-1" t-if="question.question_type == 'multiple_choice'">
-                        <span t-esc="question_data['partial_inputs_count']" class="mr-1"/>Partial
-                    </span>
+            <div>
```

Comment:
I mainly changed the margins of the badges, but the wrapping div is there so that the badges do not overflow out of the screen on mobile devices: they now go on the next line